examples/libs/ethersjs/v6/hardhat-deploy/.env.example

```
@@ -0,0 +1,10 @@
+DFNS_API_URL='https://api.dfns.ninja'
+DFNS_APP_ID='<redacted>'
+DFNS_APP_ORIGIN='http://localhost:3000'
+DFNS_CRED_ID='<redacted>'
+DFNS_PRIVATE_KEY='<redacted>'
+DFNS_AUTH_TOKEN='<redacted>'
+POLYGON_MUMBAI_WALLET_ID='<redacted>'
+POLYGON_MUMBAI_PROVIDER_URL='https://rpc.ankr.com/polygon_mumbai'
+ALCHEMY_API_KEY='<redacted>'
+ETHERSCAN_API_KEY='<redacted>'
```

```suggestion
DFNS_API_URL='https://api.dfns.ninja'
DFNS_APP_ID='ap-gcnp5...<redacted>...'
DFNS_APP_ORIGIN='http://localhost:3000'
DFNS_CRED_ID='Y2kt...<redacted>...'
DFNS_PRIVATE_KEY='-----BEGIN RSA PRIVATE KEY-----
MIIEogIBAAKCAQEAxWRXN5iLhFyIucblxPY7zjbSKc3qpjcnF9g2pdN+HcMhH+hK
Q3wcpOIswNuCxBRm39SHJdlNsWx5lKdEKgml7uPDxO2YoU8kbyygkWczxfzSDlRh
... <redacted> ...
WsLi69hdaicjD72EXWK2+dfKsBZNdx2UwFm3b0OtptAZzE8cV0MHh26iEgerlkcH
lVoZStj5fsYOQlKmQJl6JHTOARAHmk8OL4LyFDI9dp8V15UqXgo=
-----END RSA PRIVATE KEY-----'
DFNS_AUTH_TOKEN='eyJ0...<redacted>...'

POLYGON_WALLET_ID='<redacted>'
POLYGON_NODE_URL='https://polygon-mumbai-bor.publicnode.com'
```

a couple minor things. we standardized the sample env vars across all examples. and all examples when possible uses public nodes, so they can easily run without requiring the devs to go register with specific sites.

one additional though is etherscan contract verification is optional right? I think we should skip that and make the example as streamlined as possible.

---

Agreed, the Etherscan part came off from the original `hardhat` example but I don't think is that important.